test/routes/token.js

```diff
@@ -139,17 +170,6 @@ describe('/token POST', function () {
       });
     });
 
-    it('forbids client_secret', (done) => {
-      v({
-        client_secret: CLIENT_SECRET,
-        code: CODE,
-        grant_type: GRANT_JWT,
-      }, (err) => {
-        joiNotAllowed(err, 'client_secret');
-        done();
-      });
-    });
-
```

Comment:
This test was an exact duplicate of the one immediately above it, so I took the opportunity to remove it.